server/src/machines/extruder1/mod.rs

```diff
@@ -0,0 +1,333 @@
+use api::{ExtruderV2Events, ExtruderV2Namespace};
+use control_core::{
+    actors::{
+        analog_input_getter::AnalogInputGetter,
+        mitsubishi_inverter_rs485::{
+            MitsubishiControlRequests, MitsubishiInverterRS485Actor, MitsubishiModbusRequest,
+        },
+    },
+    machines::Machine,
+    socketio::namespace::NamespaceCacheingLogic,
+};
+
+use pressure_controller::PressureController;
+use serde::{Deserialize, Serialize};
+use std::time::Instant;
+use temperature_controller::TemperatureController;
+pub mod act;
+pub mod api;
+pub mod new;
+pub mod pressure_controller;
+pub mod temperature_controller;
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+pub enum ExtruderV2Mode {
+    Standby,
+    Heat,
+    Extrude,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+pub struct Heating {
+    pub temperature: f32,
+    pub heating: bool,
+    pub target_temperature: f32,
+    pub wiring_error: bool,
+}
+
+impl Default for Heating {
+    fn default() -> Self {
+        Self {
+            temperature: 0.0,
+            heating: false,
+            target_temperature: 150.0,
+            wiring_error: false,
+        }
+    }
+}
+
+pub enum HeatingType {
+    Nozzle,
+    Front,
+    Back,
+    Middle,
+}
+
+#[derive(Debug)]
+pub struct ExtruderV2 {
+    inverter: MitsubishiInverterRS485Actor,
+    namespace: ExtruderV2Namespace,
+    mode: ExtruderV2Mode,
+    last_measurement_emit: Instant,
+    pressure_sensor: AnalogInputGetter, // EL3024
+    uses_rpm: bool,
+    bar: f32,
+    target_rpm: f32,
+    target_bar: f32,
+    temperature_controller_front: TemperatureController,
+    temperature_controller_middle: TemperatureController,
+    temperature_controller_back: TemperatureController,
+    temperature_controller_nozzle: TemperatureController,
+    pressure_motor_controller: PressureController,
+}
```

Comment:
Why is `bar`, `target_rpm`, `target_bar` just lying around. Wrap them in a controller.

And not `PressureController`.... Make a `ScrewSpeedController` that handles all the targets and regulation mode.